ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -28,9 +29,10 @@ import (
 )
 
 const (
-	RayJobDefaultRequeueDuration    = 3 * time.Second
-	RayJobDefaultClusterSelectorKey = "ray.io/cluster"
-	PythonUnbufferedEnvVarName      = "PYTHONUNBUFFERED"
+	RayJobDefaultRequeueDuration        = 3 * time.Second
+	RayJobDefaultClusterSelectorKey     = "ray.io/cluster"
+	PythonUnbufferedEnvVarName          = "PYTHONUNBUFFERED"
+	DELETE_RAYJOB_CR_AFTER_JOB_FINISHES = "DELETE_RAYJOB_CR_AFTER_JOB_FINISHES"
```

Comment:
Consider adding this to Config API too: https://github.com/ray-project/kuberay/tree/master/ray-operator/apis/config/v1alpha1 

Not a blocker since it's still alpha

Reply:
Done